resources/assets/coffee/react/profile-page/medals.coffee

```diff
@@ -32,6 +33,15 @@ ProfilePage.Medals = React.createClass
 
     _.includes @_achieved, id
 
+  _getDateAchieved: (id) ->
```

Comment:
as in:

```
{
    "achieved_at": "2015-07-15T20:27:39+0000",
    "achievement": {
        "data": {
            "id": 13,
            "name": "Catch 20,000 fruits",
            "grouping": "Dedication",
            "ordering": 1,
            "slug": "fruits-hits-20000",
            "description": "That is a lot of dietary fiber."
        }
    }
}
```
?

but then the achievement is wrapped with another object having the `achieved_at` value. it would be kinda painful to implement for the badge to accept both that and the normal locked achievement